forteConfig documentation

SamueleA opened the pull request:
documentation for the `forteConfig` object.

SamueleA committed:
forteConfig documentation

SamueleA committed:
Merge ac4911e3664e685d1b656d98b92f6d865dfc3fed into b625783a24217ec35446ae28cc47a7a36e6f8703

github-actions[bot] committed:
chore(i18n): update translations [en]

Sync file structure, format locales. Branch: 199/merge

mreg-17 commented on sdk/web/checkout-sdk/hooks/useSelectPaymentModal.mdx:
```diff
@@ -176,6 +176,58 @@ The `settings` object can include the following properties:
+
+The forteConfig object must be provided to enable Forte payments.
+
+The protocol must be set to either `custom_evm_call` and or `mint` depending on whether the transaction is a mint or a general transaction.
```
Probably should be OR instead of AND OR?

SamueleA replied:
this has been fixed in commit [20dc6b3](https://github.com/0xsequence/docs/pull/199/commits/20dc6b332b628b222633994f31f196e27e2b26a4)

mreg-17 commented on sdk/web/checkout-sdk/hooks/useSelectPaymentModal.mdx:
```diff
@@ -176,6 +176,58 @@ The `settings` object can include the following properties:
+
+#### forteConfig
+
+The forteConfig object must be provided to enable Forte payments.
```
`forteConfig` for consistency

mreg-17 commented on sdk/web/checkout-sdk/hooks/useSelectPaymentModal.mdx:
```diff
@@ -176,6 +176,58 @@ The `settings` object can include the following properties:
+The protocol must be set to either `custom_evm_call` and or `mint` depending on whether the transaction is a mint or a general transaction.
+Furthermore, a `sellerAddress` string must be provided, indicating the address of the seller in the case of a transaction or the address of the contract in the case of a mint
+
+The calldata can be either string with the calldata (the same as the `txData` already provided) or a structured calldata object, detailing the function name and the arguments. Crucially, the structured calldata requires a receiver address to be specified as shown in the example below.
```
either "a" string with ...

SamueleA replied:
this has been fixed in commit [20dc6b3](https://github.com/0xsequence/docs/pull/199/commits/20dc6b332b628b222633994f31f196e27e2b26a4)

mreg-17 commented on sdk/web/checkout-sdk/hooks/useSelectPaymentModal.mdx:
```diff
@@ -176,6 +176,58 @@ The `settings` object can include the following properties:
+The forteConfig object must be provided to enable Forte payments.
+
+The protocol must be set to either `custom_evm_call` and or `mint` depending on whether the transaction is a mint or a general transaction.
+Furthermore, a `sellerAddress` string must be provided, indicating the address of the seller in the case of a transaction or the address of the contract in the case of a mint
```
missing period at the end.

SamueleA replied:
this has been fixed in commit [20dc6b3](https://github.com/0xsequence/docs/pull/199/commits/20dc6b332b628b222633994f31f196e27e2b26a4)

0xsequence-bot committed:
[AUTOMATED] Update stack/api OpenAPI docs (#84)

* [AUTOMATED] Update: api/proto/docs/api.gen.yaml

* [AUTOMATED] Update: api/proto/docs/api.gen.yaml

* [AUTOMATED] Update: api/proto/docs/api.gen.yaml

* [AUTOMATED] Update: api/proto/docs/api.gen.yaml

* [AUTOMATED] Update: api/proto/docs/api.gen.yaml

* [AUTOMATED] Update: api/proto/docs/api.gen.yaml

* [AUTOMATED] Update: api/proto/docs/api.gen.yaml

* [AUTOMATED] Update: api/proto/docs/api.gen.yaml

* [AUTOMATED] Update: api/proto/docs/api.gen.yaml

* [AUTOMATED] Update: api/proto/docs/api.gen.yaml

* [AUTOMATED] Update: api/proto/docs/api.gen.yaml

* [AUTOMATED] Update: api/proto/docs/api.gen.yaml

* [AUTOMATED] Update: api/proto/docs/api.gen.yaml

* [AUTOMATED] Update: api/proto/docs/api.gen.yaml

* [AUTOMATED] Update: api/proto/docs/api.gen.yaml

* [AUTOMATED] Update: api/proto/docs/api.gen.yaml

* [AUTOMATED] Update: api/proto/docs/api.gen.yaml

* [AUTOMATED] Update: api/proto/docs/api.gen.yaml

* [AUTOMATED] Update: api/proto/docs/api.gen.yaml

* [AUTOMATED] Update: api/proto/docs/api.gen.yaml

* [AUTOMATED] Update: api/proto/docs/api.gen.yaml

* [AUTOMATED] Update: api/proto/docs/api.gen.yaml

* Auto-update guide cards metadata [skip ci]

---------

Co-authored-by: pkieltyka <18831+pkieltyka@users.noreply.github.com>
Co-authored-by: taylanpince <12650+taylanpince@users.noreply.github.com>
Co-authored-by: david-littlefarmer <17728576+david-littlefarmer@users.noreply.github.com>
Co-authored-by: nachopiris <66640666+nachopiris@users.noreply.github.com>
Co-authored-by: SamueleA <26363061+SamueleA@users.noreply.github.com>
Co-authored-by: VojtechVitek <139342+VojtechVitek@users.noreply.github.com>
Co-authored-by: shunkakinoki <39187513+shunkakinoki@users.noreply.github.com>
Co-authored-by: github-actions[bot] <github-actions[bot]@users.noreply.github.com>

andygruening committed:
Added more details on recovering wallet from storage and smart contract return types (#201)

* added more details on recovering wallet from storage

* chore(i18n): update translations [en]

Sync file structure, format locales. Branch: 201/merge

* explanation for return types

* chore(i18n): update translations [en]

Sync file structure, format locales. Branch: 201/merge

---------

Co-authored-by: github-actions[bot] <github-actions[bot]@users.noreply.github.com>

SamueleA committed:
remove mentions of useERC1155SaleContractCheckout due to deprecation (#202)

* remove mentions of erc 1155 utility hook due to deprecation

* chore(i18n): update translations [en]

Sync file structure, format locales. Branch: 202/merge

---------

Co-authored-by: github-actions[bot] <github-actions[bot]@users.noreply.github.com>

0xsequence-bot committed:
[AUTOMATED] Update files from 0xsequence/marketplace-api (#200)

* [AUTOMATED] Update: proto/docs/marketplace.gen.yaml

* [AUTOMATED] Update: proto/docs/marketplace.gen.yaml

* [AUTOMATED] Update: proto/docs/marketplace.gen.yaml

* [AUTOMATED] Update: proto/docs/marketplace.gen.yaml

* [AUTOMATED] Update: proto/docs/marketplace.gen.yaml

* [AUTOMATED] Update: proto/docs/marketplace.gen.yaml

* [AUTOMATED] Update: proto/docs/marketplace.gen.yaml

---------

Co-authored-by: VojtechVitek <139342+VojtechVitek@users.noreply.github.com>
Co-authored-by: david-littlefarmer <17728576+david-littlefarmer@users.noreply.github.com>

0xsequence-bot committed:
[AUTOMATED] Update files from 0xsequence/marketplace-api (#203)

* [AUTOMATED] Update: proto/docs/marketplace.gen.yaml

* [AUTOMATED] Update: proto/docs/marketplace.gen.yaml

* [AUTOMATED] Update: proto/docs/marketplace.gen.yaml

* [AUTOMATED] Update: proto/docs/marketplace.gen.yaml

* [AUTOMATED] Update: proto/docs/marketplace.gen.yaml

* [AUTOMATED] Update: proto/docs/marketplace.gen.yaml

---------

Co-authored-by: VojtechVitek <139342+VojtechVitek@users.noreply.github.com>
Co-authored-by: david-littlefarmer <17728576+david-littlefarmer@users.noreply.github.com>

0xsequence-bot committed:
[AUTOMATED] Update relayer OpenAPI docs (#187)

* [AUTOMATED] Update: proto/docs/relayer.gen.yaml

* [AUTOMATED] Update: proto/docs/relayer.gen.yaml

* [AUTOMATED] Update: proto/docs/relayer.gen.yaml

* [AUTOMATED] Update: proto/docs/relayer.gen.yaml

* [AUTOMATED] Update: proto/docs/relayer.gen.yaml

---------

Co-authored-by: pkieltyka <18831+pkieltyka@users.noreply.github.com>
Co-authored-by: klaidliadon <5322228+klaidliadon@users.noreply.github.com>

andygruening committed:
Unity/Unreal Ecosystem Wallet (#193)

* started unity restructure for wallet separation

* chore(i18n): update translations [en]

Sync file structure, format locales. Branch: 193/merge

* started unreal wallet separation

* further unity/unreal restructure

* chore(i18n): update translations [en]

Sync file structure, format locales. Branch: 193/merge

* more unity/unreal restructure

* chore(i18n): update translations [en]

Sync file structure, format locales. Branch: 193/merge

* added new unity/unreal boilerplate images

* latest unity/unreal updates

* chore(i18n): update translations [en]

Sync file structure, format locales. Branch: 193/merge

* added unreal warning for different wallet class names between v2/v3

* new unreal ecosystem blockchain interaction structure

* removed "Universal" as a section in Unitys bootstrap docs

* fixed broken links

* chore(i18n): update translations [en]

Sync file structure, format locales. Branch: 193/merge

* fixed broken links

* chore(i18n): update translations [en]

Sync file structure, format locales. Branch: 193/merge

* fixed broken links

* chore(i18n): update translations [en]

Sync file structure, format locales. Branch: 193/merge

* fixed broken links

* chore(i18n): update translations [en]

Sync file structure, format locales. Branch: 193/merge

* fixed broken links

* chore(i18n): update translations [en]

Sync file structure, format locales. Branch: 193/merge

* Auto-update guide cards metadata [skip ci]

* fixed session recovery from previous master merge

* chore(i18n): update translations [en]

Sync file structure, format locales. Branch: 193/merge

* added info for unity embedded wallet boilerplates

* added unreal migration docs

* chore(i18n): update translations [en]

Sync file structure, format locales. Branch: 193/merge

* latest unreal fixes

* chore(i18n): update translations [en]

Sync file structure, format locales. Branch: 193/merge

* some changes to re-run the failed test

* fixed broken unreal migration link

* chore(i18n): update translations [en]

Sync file structure, format locales. Branch: 193/merge

---------

Co-authored-by: github-actions[bot] <github-actions[bot]@users.noreply.github.com>

0xsequence-bot committed:
[AUTOMATED] Update: proto/docs/relayer.gen.yaml (#204)

Co-authored-by: pkieltyka <18831+pkieltyka@users.noreply.github.com>

SamueleA committed:
fixed typos

SamueleA committed:
Merge 20dc6b332b628b222633994f31f196e27e2b26a4 into 18127c79fa3ba99404163b9d1ba9eff4ae066d3d

github-actions[bot] committed:
chore(i18n): update translations [en]

Sync file structure, format locales. Branch: 199/merge
